grafana/resource_data_source.go

```diff
@@ -237,6 +252,21 @@ source selected (via the 'type' argument).
 							Optional:    true,
 							Description: "(Elasticsearch) Index date time format. nil(No Pattern), 'Hourly', 'Daily', 'Weekly', 'Monthly' or 'Yearly'.",
 						},
+						"log_analytics_client_id": {
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "(Azure Monitor) Log analytics client id.",
+						},
+						"log_analytics_default_workspace": {
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "(Azure Monitor) Log analytics default workspace.",
+						},
+						"log_analytics_tenant_id": {
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "(Azure Monitor) Log analytics tenant ID.",
+						},
```

Comment:
Seems like all log analytics values are deprecated

https://github.com/grafana/grafana/blob/d5f40b63bbafe0126086d42bde2caae59bed452b/public/app/plugins/datasource/grafana-azure-monitor-datasource/azure_log_analytics/azure_log_analytics_datasource.ts#L228-L234

Reply:
Alright, All fields of Azure Monitor with the log_analytics prefix are removed :+1: